modules/qdrant/src/main/java/org/testcontainers/qdrant/QdrantContainer.java

```diff
@@ -3,34 +3,64 @@
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.containers.wait.strategy.Wait;
 import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
 
 /**
  * Testcontainers implementation for Qdrant.
- * <p>
- * Supported image: {@code qdrant/qdrant}
- * <p>
- * Exposed ports:
+ *
+ * <p>Supported image: {@code qdrant/qdrant}
+ *
+ * <p>Exposed ports:
+ *
  * <ul>
- *     <li>HTTP: 6333</li>
- *     <li>Grpc: 6334</li>
+ *   <li>HTTP: 6333
+ *   <li>GRPC: 6334
  * </ul>
  */
 public class QdrantContainer extends GenericContainer<QdrantContainer> {
 
     private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("qdrant/qdrant");
 
+    private final int QDRANT_REST_PORT = 6333;
+
+    private final int QDRANT_GRPC_PORT = 6334;
+
+    private final String CONFIG_FILE_PATH = "/qdrant/config/config.yaml";
+
+    private final String API_KEY_ENV = "QDRANT__SERVICE__API_KEY";
+
     public QdrantContainer(String image) {
         this(DockerImageName.parse(image));
     }
 
     public QdrantContainer(DockerImageName dockerImageName) {
         super(dockerImageName);
         dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
-        withExposedPorts(6333, 6334);
-        waitingFor(Wait.forHttp("/readyz").forPort(6333));
+        withExposedPorts(QDRANT_REST_PORT, QDRANT_GRPC_PORT);
+        waitingFor(Wait.forHttp("/readyz").forPort(QDRANT_REST_PORT));
+    }
+
+    public QdrantContainer withApiKey(String apiKey) {
+        return withEnv(API_KEY_ENV, apiKey);
+    }
```

Comment:
we have been trying to avoid having specific methods for env vars due to it can be done using the container definition

```java
new QdrantContainer("qdrant/qdrant:v1.7.4")
    .withEnv("QDRANT__SERVICE__API_KEY", "abc");
```

Reply:
This PR is an effort to offer QOL improvements by adding handy methods like these to the Qdrant module.